scene/2d/polygon_2d.cpp

```diff
@@ -223,8 +223,8 @@ void Polygon2D::_notification(int p_what) {
 
 				for (int i = 0; i < bone_weights.size(); i++) {
 					if (bone_weights[i].weights.size() != points.size()) {
-						continue; //different number of vertices, sorry not using.
 						print_line("wrong weight size");
+						continue; //different number of vertices, sorry not using.
```

Comment:
I would outright remove the three `print_line` here, that's just debugging statements used while implementing it.

Reply:
Better yet, convert it to ERR_COND_COONTINUE or something like that, so that it emits an actual error.

Reply:
I'm not sure if it's actually meant to print an error at all. Those cases should be skipped, but are they a bug per se which would warrant spamming the user with errors?

Reply:
I chose to use `ERR_CONTINUE`, just to be sure.

Reply:
There is no need for `continue;` if you're using `ERR_CONTINUE`.

Reply:
@reduz confirmed that those prints can simply be removed.

Reply:
Done.